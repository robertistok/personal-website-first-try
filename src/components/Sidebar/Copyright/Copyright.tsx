import React from "react";
import styles from "./Copyright.module.scss";

interface CopyrightProps {
  copyright: string;
}

const Copyright: React.FunctionComponent<CopyrightProps> = ({
  copyright
}: CopyrightProps): React.ReactElement => <div className={styles["copyright"]}>{copyright}</div>;

export default Copyright;
