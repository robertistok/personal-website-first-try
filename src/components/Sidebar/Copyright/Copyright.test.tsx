import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import Copyright from "./Copyright";

describe("<Copyright />", () => {
  const props = { copyright: "test copyright" };

  it("renders", () => {
    shallow(<Copyright {...props} />);
  });

  it("renders and matches snapshot", () => {
    const wrapper = shallow(<Copyright {...props} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
