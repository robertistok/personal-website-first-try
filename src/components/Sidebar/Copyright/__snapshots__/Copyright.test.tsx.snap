// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Copyright /> renders and matches snapshot 1`] = `
<div
  className="copyright"
>
  test copyright
</div>
`;
